manifests/a/Amazon/SessionManagerPlugin/1.2.279.0/Amazon.SessionManagerPlugin.locale.en-US.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Amazon.SessionManagerPlugin
+PackageVersion: 1.2.279.0
+PackageLocale: en-US
+Publisher: AWS
```

```suggestion
Publisher: Amazon Web Services
```
This must match Control Panel.